### PR TITLE
DAT-2760 Backwards compatibility in <image> tag for portable infoboxes

### PR DESCRIPTION
**Pull request by sebastianmarzjan**

### CODE REVIEW

**Comment by jacekjursza** [comment]
LGTM

**Comment by SebastianMarzjan** [comment]
Merging to feature branch

File: extensions/wikia/PortableInfobox/services/Helpers/ImageFilenameSanitizer.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Wikia\PortableInfobox\Helpers;
+
+class ImageFilenameSanitizer {
```
Comment (wladekb) [inline extensions/wikia/PortableInfobox/services/Helpers/ImageFilenameSanitizer.php]: It looks like a general-interest functionality. Maybe we have an existing implementation somewhere else or we should put it in some common place?

Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/services/Helpers/ImageFilenameSanitizer.php]: Will check on next iteration on this, thanks for the input!

**Pull request merged**